packages/wp-now/README.md

```diff
@@ -4,27 +4,101 @@
 
 ## Getting Started
 
+Before getting started, make sure you have `nvm` installed. If you need to install it first,
+[follow these installation instructions](https://github.com/nvm-sh/nvm#installation).
+
 Follow these steps to build and run `wp-now` locally:
 
 ### Building
 
 To build the project, use the following commands in your terminal:
 
+1. Clone this repository locally
+2. Execute `nvm use`
+3. Execute `yarn install` followed by `yarn build`
+
+### Running
+
+1. Execute `sudo npm install -g nx@latest`. This will make `nx` command globally accessible in the project.
+2. To start the web server in the plugin or theme mode (see Modes on WP-NOW section for details), run the command below. Make sure to replace `/path/to/wordpress-plugin-or-theme` with the actual path to your plugin or theme folder.
+
 ```bash
-nvm use
-yarn install
-yarn build
+nx preview wp-now start --path=/path/to/wordpress-plugin-or-theme
 ```
 
-### Running
+### Modes on WP-NOW
 
-To start the web server and execute your WordPress plugin or theme, use the following command:
+wp-now operates in four different modes:
+
+-   `index`: executes a simple php file
+-   `theme`: loads WordPress with your selected theme included
+-   `plugin`: loads WordPress with your selected plugin included
+-   `wp-content`: loads WordPress site that contains plugins and themes from the provided wp-content directory
+
+To launch the `wp-now` in the `index`, `theme` or `plugin` mode, you can run:
 
 ```bash
 nx preview wp-now start --path=/path/to/wordpress-plugin-or-theme
```

Comment:
As https://github.com/WordPress/wordpress-playground/pull/272 is merged, should we encourage running local environments from the project directory context?

```
cd wordpress-plugin-or-theme
wp-now start
```